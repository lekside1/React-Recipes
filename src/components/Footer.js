import React from 'react';
import HyperLink from './containers/HyperLink';

const Footer = () => (
  <footer>
    <div className="mt3">
      <div className="phantom" />
      <div className="footer">
        <HyperLink to="/" activeClassName="bg-black">Welcome</HyperLink>
        <HyperLink to="/about" activeClassName="bg-black">About</HyperLink>
      </div>
    </div>
  </footer>
);

export default Footer;
