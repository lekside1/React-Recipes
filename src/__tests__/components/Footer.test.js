import React from 'react';
import renderer from 'react-test-renderer';
import { BrowserRouter } from 'react-router-dom';
import Footer from '../../components/Footer';

it('Should Render Header', () => {
  const tree = renderer.create(
    <BrowserRouter>
      <Footer />
    </BrowserRouter>,
  ).toJSON();

  expect(tree).toMatchSnapshot();
});
