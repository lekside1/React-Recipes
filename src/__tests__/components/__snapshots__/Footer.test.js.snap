// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should Render Header 1`] = `
<footer>
  <div
    className="mt3"
  >
    <div
      className="phantom"
    />
    <div
      className="footer"
    >
      <a
        aria-current="page"
        className="p1 mx2 white rounded text-decoration-none bg-black"
        href="/"
        onClick={[Function]}
        style={Object {}}
      >
        Welcome
      </a>
      <a
        aria-current={null}
        className="p1 mx2 white rounded text-decoration-none"
        href="/about"
        onClick={[Function]}
      >
        About
      </a>
    </div>
  </div>
</footer>
`;
